chain/consensus/mir/manager.go

```diff
@@ -234,68 +235,151 @@ func NewManager(ctx context.Context, addr address.Address, h host.Host, api v1ap
 		if err != nil {
 			return nil, fmt.Errorf("failed to create interceptor: %w", err)
 		}
-		m.MirNode, err = mir.NewNode(t.NodeID(mirID), nodeCfg, smrSystem.Modules(), nil, m.interceptor)
+		m.mirNode, err = mir.NewNode(t.NodeID(mirID), nodeCfg, smrSystem.Modules(), nil, m.interceptor)
 	} else {
-		m.MirNode, err = mir.NewNode(t.NodeID(mirID), nodeCfg, smrSystem.Modules(), nil, nil)
+		m.mirNode, err = mir.NewNode(t.NodeID(mirID), nodeCfg, smrSystem.Modules(), nil, nil)
 	}
 	if err != nil {
-		return nil, fmt.Errorf("failed to create Mir node: %w", err)
+		return nil, fmt.Errorf("validator %v failed to create Mir node: %w", mirID, err)
 	}
 
 	return &m, nil
 }
 
-// Start starts the manager.
-func (m *Manager) Start(ctx context.Context) chan error {
-	log.Infof("Mir manager %s starting", m.MirID)
-	log.Info("Mir initial checkpointing period: ", m.StateManager.GetCheckpointPeriod())
+func (m *Manager) Serve(ctx context.Context) error {
```

Comment:
Why `Serve`? is less obvious than `Start`, do you mind adding a note on what this function does?

Reply:
For me, `Start` is not a blocking call. Before, we called `Start` and then were blocked in Mine's `for` loop. Now we don't have that loop in Mine. That loop in all logic is inside Server `function.` 
We could name it `Mine` but do not have miners in Mir.